### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by 0x96e63** [kaJob-dev/growtopia.py#16]
Can you push a complete guide to each function to operate the library?
A directory named 'documentation' containing files of co responding files may be useful,
or, a simple iterative documentation on the readme covering each function can be useful.

**Comment by kaJob-dev** [comment]
Hiya, thanks for pointing this out :-)

I've been putting some examples out on the discord server, though I do understand that some people would rather not join or might just not use discord. I'll be working on the repo's Wiki soon (probably this week). 

As for now.. I can probably provide you with some of the examples posted on the discord server.

**Comment by Bt08s** [comment]
It will be so good if you finish this project

**Comment by kaJob-dev** [comment]
> It will be so good if you finish this project

Yeah, will start working on it again soon (hopefully, and the docs asw 💀)

**Comment by Bt08s** [comment]
Yeah peoples can finally start to code stuffs in gt with python. Is it possible to read gamelogs and change them?
